module-3/back-end/src/models/project.js

```diff
@@ -10,10 +10,8 @@ const Model = (config) => {
     const create = async project => {
 
         return await knex(config.tableName).insert(project, [
-            'id',
             'title',
```

Comment:
Why an `id` has been removed? Don't we need to know a project id in the response?